lib/content-services/src/lib/legal-hold/services/legal-hold.service.ts

```diff
@@ -87,4 +87,26 @@ export class LegalHoldService {
     unassignHold(holdId: string, holdChildId: string): Observable<undefined> {
         return from(this.legalHoldApi.unassignHold(holdId, holdChildId));
     }
+
+    /**
+     * Create hold.
+     *
+     * @param filePlanId The identifier of a file plan. You can also use the -filePlan- alias.
+     * @param holds Holds to create
+     * @returns List of created holds Observable<HoldEntry[]>
+     */
+    createHold(filePlanId: string, holds: Hold[]): Observable<HoldEntry> {
```

Comment:
Also what about the docs for the service?

Reply:
Added